### PR TITLE
fix: add shim for js

### PR DESCRIPTION
**Pull request by paralin**
Fixes build with:

env GOOS=js GOARCH=wasm go build -trimpath .

### CODE REVIEW

**Comment by AkihiroSuda** [comment]
How is this expected to be used?

**Comment by paralin** [comment]
It's a shim so that programs built using the console package can compile against GOOS=js as well.

**Comment by dmcgowan** [comment]
@paralin can you add a little bit more context to this issue. We don't currently have any ongoing efforts to support `GOOS=js` or issues tracking it. Knowing more about the use case would be helpful.

**Comment by paralin** [comment]
@dmcgowan The bubbletea TUI packages import containerd/console. I recently submitted several PRs to projects adding shims so that the build will complete if GOOS=js and GOARCH=wasm with reasonable defaults.

I think it makes sense in general to make the package at least build cleanly with that GOOS. Even if the functions are all shims.

**Comment by dmcgowan** [comment]
@paralin if this library is being used for a platform other than what is explicitly supported, our normal pattern is to include a `console_other.go` file which builds with `!` statements of the supported platforms. The current change in `console_js.go` aren't specific to any platform.

**Comment by paralin** [comment]
@dmcgowan okay, applied the change.

File: console_other.go [hunk 1]
```diff
@@ -0,0 +1,35 @@
+// +build !darwin,!freebsd,!linux,!netbsd,!openbsd,!solaris
```
Comment (AkihiroSuda) [inline console_other.go:1]: Please gofmt with Go 1.17

**Review by AkihiroSuda — CHANGES_REQUESTED** [review]

File: console_other.go (hunk 1 above)
Comment (paralin) [reply console_other.go:1]: @AkihiroSuda If I do that, the license checker will fail. I previously sent the PR with it formatted that way, and the test failed. The containerd license checker expects this format and doesn't allow the other one.

**Comment by paralin** [comment]
As mentioned in the conversation - the license checker fails if formatted with go 1.17

File: console_other.go (hunk 1 above)
Comment (paralin) [reply console_other.go]:
```
    - name: Check file headers
      run: ../project/script/validate/fileheader ../project/
      working-directory: src/github.com/containerd/console

```

**Comment by paralin** [comment]
@thaJeztah @AkihiroSuda Fixed & rebased, apologies for the delay!

**Review by AkihiroSuda — APPROVED** [review]

**Pull request merged**